assets/src/edit-story/utils/useIntersectionEffect.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useEffect } from 'react';
+import IntersectionObserver from 'intersection-observer-polyfill';
```

Comment:
I am not sure we support ie11 in this project. Other than that broswer, do we need this. 

CC @pbakaus

Reply:
For now, I followed [Dima](https://github.com/google/web-stories-wp/blame/c29f388b8135b9b20e061d036c01222f8fa10cbb/assets/src/edit-story/utils/useResizeEffect.js#L20), we can remove it anytime we agree on ever-green browsers.